recipes/libgeoip/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "GeoIP" %}
+{% set version = "1.6.11" %}
+{% set sha256 = "8859cb7c9cb63e77f4aedb40a4622024359b956b251aba46b255acbe190c34e0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/maxmind/geoip-api-c/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - gcc
```

Comment:
If you need the `gcc` package you'll need the `libgcc` at run time. But I suspect you don't really need `gcc` here. CAn you try removing it?